tests/by-util/test_wc.rs

```diff
@@ -464,3 +464,41 @@ fn test_files0_from_with_stdin_try_read_from_stdin() {
         .stderr_contains(MSG)
         .stdout_is("");
 }
+
+#[cfg(unix)]
+#[test]
+fn test_bash() {
+    let wc_path = path_concat!(env!("OUT_DIR"), "..", "..", "..", "coreutils");
+
+    let scene = TestScenario::new(util_name!());
+
+    let first = scene
+        .cmd("bash")
+        .arg("-c")
+        .arg(format!(
+            "(dd ibs=1 skip=0 count=0; {} wc -c) < lorem_ipsum.txt",
+            wc_path
+        ))
+        .succeeds()
+        .stdout_str()
+        .lines()
+        .last()
+        .unwrap()
+        .parse::<i32>()
+        .unwrap();
+    let second = scene
+        .cmd("bash")
+        .arg("-c")
+        .arg(format!(
+            "(dd ibs=1 skip=3 count=0; {} wc -c) < lorem_ipsum.txt",
+            wc_path
+        ))
+        .succeeds()
+        .stdout_str()
+        .lines()
+        .last()
+        .unwrap()
+        .parse::<i32>()
+        .unwrap();
+    assert!(first > second);
```

Comment:
Is it possible to make it more precise or does that not work? I.e. give the exact values that should be returned?

Reply:
Skimming through the dd manual and found `status`. But still need to trim because it prints a trailing newline.

```
status=LEVEL
              The LEVEL of information to print to stderr; 'none'
              suppresses everything but error messages, 'noxfer'
              suppresses the final transfer statistics, 'progress' shows
              periodic transfer statistics
```